extensions/interactions/GraphInput/protractor.js

```diff
@@ -57,11 +57,11 @@ var submitAnswer = async function(conversationInput, graphDict) {
 
 var answerObjectType = 'Graph';
 var initialGraph = {
-  vertices: [[277, 77], [248, 179], [405, 144]]
+  vertices: [[-20, 50], [10, 60], [40, -20]]
 };
 var responseGraphDict = {
   edges: [[0, 1], [1, 2], [0, 2]],
-  vertices: [[277, 77], [248, 179], [405, 144]]
+  vertices: [[-20, 50], [10, 60], [40, -20]]
```

Comment:
Why these changes? I'm not seeing what they have to do with the upgrade

Reply:
Ah, ok so these are used as co-ordinates for testing the GraphInput interaction. See code below:

https://github.com/oppia/oppia/blob/3a5082be27ee77ceb748992fe2e901d7571a9d29/core/tests/protractor_utils/forms.js#L56-L57

With the upgrade, these offset values seem to be outside the container for the Graph input container. So I changed this a bit for it to work (basically made the vertices less sparse in the graph).